### DISCUSSION TITLE
Planning application data specifications

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#94] [Planning consideration]
# Planning application data specifications

## Why are we looking at this

Planning applications are a crucial part of the planning system.

Lots goes into making a planning application. Lots of work is required to make a decision on it. And, lots of work is done to monitor the progress of it.

Essentially, there is a lot to the "planning application" space.

Therefore, one of our primary tasks is to understand all the data specifications that can support planning applications. Once identified we can work through them one by one.

By implementing a set of consistent specifications throughout the whole planning application system the whole process should become smoother, faster and more efficient.

## What is this discussion for

We are looking for help to identify and define the different components of "planning applications" that can be improved with data specifications.

If you have any thoughts please share them with us.

As we identify specific data specifications to work on we will start discussion threads for the specifics about those.

## Identified data specifications

The components we think might warrant a data specification include:

* [planning application submissions](https://github.com/digital-land/data-standards-backlog/discussions/98)
* [planning application decisions](https://github.com/digital-land/data-standards-backlog/discussions/25)
* [planning conditions](https://github.com/digital-land/data-standards-backlog/discussions/40)
* planning application types

**Comment by MichaelLondonSF** [comment]
I am a veteran of London community organisations in the planning field, currently most active in Justspace.org.uk and editor of their web site. I (and many of our member groups) are active users of LPA web sites and of the mayor of London's PlanApps (for which I was a test-user).  I would be very keen to ensure that the needs of community organisations are fully covered by the work you are doing. I am already a member of GitHub. m.edwards@ucl.ac.uk

**Reply by colmjude** [reply]
Hi @MichaelLondonSF - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Reply by cday-hybrid** [reply]
Hi colmjude, I have just registered for the Data Standards Community: Planning Permissions meeting tomorrow by following the link (https://forms.office.com/Pages/ResponsePage.aspx?id=EGg0v32c3kOociSi7zmVqN_GMPIIIgpJg3NuRnb3B4dUNzVROFVXVzhQU1pZTEk1TVpNVlhOVkZYRy4u), but have not received an automated calendar invite?  Can you help?  Thanks, Claire - Hybrid Planning & Development.

**Reply by jamesatervit** [reply]
> Hi colmjude, I have just registered for the Data Standards Community: Planning Permissions meeting tomorrow by following the link (https://forms.office.com/Pages/ResponsePage.aspx?id=EGg0v32c3kOociSi7zmVqN_GMPIIIgpJg3NuRnb3B4dUNzVROFVXVzhQU1pZTEk1TVpNVlhOVkZYRy4u), but have not received an automated calendar invite? Can you help? Thanks, Claire - Hybrid Planning & Development.

same claire, no invites from the meeting on my side either.

**Comment by James-Planningportal** [comment]
Hi, I work at Planning Portal and we have an established interest in this work and in understanding the requirements and data standards MHCLG will be establishing. We have already been engaged with policy and digital teams in regard to this, alongside as part of our work in owning/operating the channel that currently provides ~90% of planning applications submissions online to local authorities in England and Wales as standardised data.

For context, we currently operate using: existing legacy XML and current JSON data standards that covers all question fields and metadata for every type of planning consent we provide online (including a geoJSON standard for site boundary data), and corresponding legacy/current API specifications that all England and Wales planning authorities use to integrate with us and retrieve applications submissions as data into their development management IT systems. On Github or can be contacted at james.garrett@planningportal.co.uk

**Reply by colmjude** [reply]
Thanks @James-Planningportal - great to hear you want to be a part of the working group! We'll send an invite once the plans for the working group have been finalised.

**Comment by nwalley** [comment]
Hello!  I Chair the Residential Logbook Association - trade body for the UK providers of **digital building logbooks** for the residential sector.  We are keen to be included in this project.  Our interest is ensuring that digitally enabled homeowners are able to submit planning applications, receive decisions and access planning/BC records from within their logbooks.   My contact email is: chair@rlba.org.uk 
More info on :  http://www.rlba.org.uk

**Reply by colmjude** [reply]
Hi @nwalley - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Comment by Pobbs** [comment]
Hi there. I am the Business Manager for the development management and building control service at Stockport Council. My role is primarily operations and improvement projects. I am investigating ways to ensure we have the right work in the right hands, reduce manual input wherever possible, improve the customer experience and streamline our processes. I am a member of a newly formed AI group within the council to understand where we can adopt AI to support the same. My goal is to automate basic planning applications ( potentially telecoms permitted development applications ) or similar where the application is a procedural approach and doesn't require subjective policy intervention. An observation within local authority task handling is that much resource is caught up in the onerous keying of data and there is an opportunity for efficiencies to be made. My contact email is andrea.murphy@stockport.gov.uk

**Reply by colmjude** [reply]
Hi @Pobbs - thanks for registering your interest in the working group. We'll send an invite once the final plans have been sorted. Your work around automating non-subjective planning applications sounds very interesting - we'd love to know more. Specifically, which planning application types or project/proposal types might be able to be automated.

**Reply by Pobbs** [reply]
I'd like to invite another industry matter expert to the group  - how can the invite be sent out ?

**Comment by Euantype** [comment]
Hello, I run and proptech company called Blocktype, where we help property developers make decisions about what to build where.  We would love to help you open up planning application data, as this would help increase certainty for our users.  :-)

**Reply by JOKane123** [reply]
Hi Euan, if you could drop me your email address I can look at including you in the work moving forward. Thanks!

**Reply by Euantype** [reply]
hi @JOKane123 i'm on euan@blocktype.co.uk

**Comment by nikkiwebber** [comment]
Hello - I work for the City of London Corporation as the Digital Planning Team Lead. We're Idox users, and have a programme of digital planning work we're undertaking. We'd really appreciate being involved in this working group - email is nicola.webber@cityoflondon.gov.uk

**Reply by colmjude** [reply]
Thanks @nikkiwebber - great you are interesting in the working group. We'll be in touch shortly once plans for the working group have been finalised.

**Comment by ArcusRobin** [comment]
Hello, Arcus provide solutions to a range of Local Authorities across the UK and would like to look at how we can work to share planning information across the community. robin.barber@arcusglobal.com

**Reply by colmjude** [reply]
Hi @ArcusRobin - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Comment by DRossiter87** [comment]
Afternoon.  I work for BSI, the UK's National Standards Body.  We have a committee dedicated to data in the construction space [B/555](https://standardsdevelopment.bsigroup.com/committees/50061658).  As part of this committee's work, we curate standards on things like [Classification](https://knowledge.bsigroup.com/products/building-construction-organization-of-information-about-construction-works-framework-for-classification?version=standard), Construction-related datasets like [IFC](https://ifc43-docs.standards.buildingsmart.org/), [data dictionaries](https://knowledge.bsigroup.com/products/building-information-modelling-and-other-digital-processes-used-in-construction-methodology-to-describe-author-and-maintain-properties-in-interconnected-data-dictionaries?version=standard), and [document metadata](https://knowledge.bsigroup.com/products/document-management-metadata-elements-and-information-reference-model?version=standard).  Many of which are known and well used within the sector.

I would be interested in supporting the work and ensuring it is aligned with the good practice we curate.

**Reply by JOKane123** [reply]
Hi Dan, if you could drop me your email address I can look at including you in the work moving forward. Thanks!

**Reply by DRossiter87** [reply]
dan.rossiter@bsigroup.com, thanks Joe!

**Comment by JHil1ier** [comment]
Having supported MHCLG in developing a digital "kit of parts" for building new houses, I see potential for digital planning to enhance efficiency and catalyse digitisation throughout the development lifecycle and value chain. 
In addition to the stated ambition, I think this initiative has the potential to catalyse a much wider  transformation. 
There are some really strong volunteers on this thread but I would be keen to join the working group, as part of a multidisciplinary team that supports a connected, holistic solution, ensuring the planning application process becomes smoother, faster, and more efficient.
My email details are: jamie.hillier@akerlof.co.uk

**Reply by colmjude** [reply]
Thanks @JHil1ier! We'll reach out when plans for the working group have been finalised.

**Comment by Ruphandley** [comment]
I’m a principle development manager at Westminster council. I am working with our digital transformation manager to improve processes and outcomes and would be interested in this work stream. Rhandley@westminster.gov.uk

**Reply by colmjude** [reply]
Thanks @Ruphandley - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Comment by veronica-diagonal** [comment]
Hello! I'd love to register diagonalWorks interest in joining the working group; we are a start up bringing together spatial planning and (geospatial) data science skills. We are both users of traditional gis processes and systems and developers of our own open source geospatial computation engine. We also bring nearly 15 years of planning consultancy expertise, wrangling planning application being key to this. We would love to join what looks like a really interesting and highly skilled pool of experts. My email is veronica@diagonal.works, my colleague's email is gala@diagonal.works. Many thanks,

**Reply by kieranwint** [reply]
Hi @veronica-diagonal - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Comment by Claudio-MTC** [comment]
Hello! I am an Associate at Momentum Transport Consultancy, very keen to look at better ways to prepare transport assessments and improve technical and non-technical communication of transport information! My email is claudio.borsari@momentum-transport.com

Looking forward to hear from the group!

**Reply by kieranwint** [reply]
Hi @Claudio-MTC - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Comment by 1WFR** [comment]
Hello, I am currently working on standardisation of data around construction and maintenance which should fit in with your proposed work. I am a Chartered Surveyor with 35 years property experience and have worked around the planning arena when running my own property development company, going through planning on apartment blocks of 50+ units. I would like to be involved with this please. My email is tim@data-door.com

**Reply by kieranwint** [reply]
Hi @1WFR - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Comment by harryquartermain** [comment]
Hi. Harry Quartermain here, I'm a chartered town planner from LandTech.  We provide aggregated information on planning applications across the country in our platform, LandInsight.  Getting cleaner data on planning applications across the whole country would be a massive benefit for the whole industry.  You can't manage what you can't measure.  I'd like to be involved please.

**Reply by kieranwint** [reply]
Hi @harryquartermain - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Comment by JBakerL** [comment]
Hello, I would like to be included please, on behalf of Lichfields. The Planning Portal tells Lichfields that we are one of their main users. Thank you, Jennie jennie.baker@lichfields.uk

**Reply by kieranwint** [reply]
Hi @JBakerL - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Comment by PRWhitehouse** [comment]
Another veteran, I was involved in the e-planning modernisation programme in early 2000 which set out some standards, including the national land and property gazetteer. Now focused on open data standards and spatial data standards. Happy to get involved in a subject that has formed 30 years of work.

**Reply by kieranwint** [reply]
Hi @PRWhitehouse - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Reply by kieranwint** [reply]
@PRWhitehouse Once we have finalised the details of the working group, we will want to get in contact with you. Do you have an email address we can reach you on? And also, what company is it you work for? Thanks in advance!

**Comment by Jasonavolve** [comment]
Hello everyone. We are specialists in electronic plan review, focus on streamlining the submission & collaborative review processes. We work with over 350 cities across North America & the EU. It’s really critical to also consider document standards as well as data. Cities have experienced significant efficiencies when document standards are applied at the submission stage which in turn accelerates the reviewing & revision processes. Document standards also enable a level of automation using AI to assist plan review and assessment of measurement for things such as CIL. Look forward to contributing to the initiative.

**Reply by kieranwint** [reply]
Hi @Jasonavolve - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Reply by kieranwint** [reply]
@Jasonavolve Once we have finalised the details of the working group, we will want to get in contact with you. Do you have an email address we can reach you on? And also, what company is it you work for? Thanks in advance!

**Comment by LanNguyenPhD** [comment]
Hi, I'm the Planning Digital Transformation officer from Oxford. Am very interested about this work and the scope and design of the base schema in the broader ecosystem of planning data. I've worked on the data modelling for our plan monitoring, which has planning applications at its core and am leading on transformation initiatives for planning in Oxford so am very interested in this work. lnguyen@oxford.gov.uk

**Reply by kieranwint** [reply]
Hi @LanNguyenPhD - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Comment by claireellul** [comment]
Hi - I'm a research academic and geospatial data engineer at University College London; very interested in maximising data reuse while balancing that against what needs to be stored/curated long term.   This includes issues such as how to better exploiting existing data from other sources within the planning process, if/where/how data-driven automation can drive efficiency and then reusing planning data elsewhere, particularly long term.   Some of my work focusses around GIS and BIM  (GeoBIM) interoperability and integration;  Also a member of the [EUNet4DBP](https://eu4dbp.net/).

**Reply by kieranwint** [reply]
Hi @claireellul - thanks for registering your interest. We'll reach out when plans for the working group have been finalised.

**Reply by kieranwint** [reply]
@claireellul Once we have finalised the details of the working group, we will want to get in contact with you. Do you have an email address we can reach you on? Thanks in advance!

**Comment by jo-gerulaitis** [comment]
Hi, I'm Jo, a Data Architect at The Planning Inspectorate (PINS) looking into the possibility of ingesting "appeals as data" when planning applications become eligible to appeal and come to us at the PINS. I want to make sure our data models are as aligned as they can be!

**Reply by kieranwint** [reply]
Hi @jo-gerulaitis - thanks for registering your interest. So that we can reach out when the plans for the working group have been finalised, what is an email address we'd be able to reach you on? Thanks in advance!

**Reply by jo-gerulaitis** [reply]
first.last[at]planninginspectorate.gov.uk

**Comment by GrahamBidwell** [comment]
Hello, I'm Graham Bidwell from OfficeLabs. We specialise in information management and digital transformation, with a strong focus on data governance, compliance, and automation solutions. We’ve supported a wide range of organisations in optimising data workflows to enhance productivity and security. We’re keen to join this working group to bring our expertise in streamlining data handling processes, and to contribute to developing scalable, efficient planning data specifications.

**Reply by kieranwint** [reply]
Hi @GrahamBidwell - thanks for registering your interest. So that we can reach out when the plans for the working group have been finalised, is there an email address we'd be able to reach you on? Thanks in advance!

**Reply by GrahamBidwell** [reply]
Thank you, email address is, graham@officelabs.co.uk.

**Comment by wps113** [comment]
Hi there, I was recommended to join this group by several different LPA digital planners as I'm in the discovery phase of building a startup to help planning officers focus on the discretionary parts of their role and reduce their workloads. 

I've seen the planning process from a number of angles through my personal experience, both here in the UK (with RE developer Crosstree and RE Investor Starwood Capital) and the US (where I worked for a permitting startup, Pulley), as well as government (Miami Dade County). Would love to help if my experience across tech and real estate and passion for addressing our housing shortage, can be useful as you put this group together.

Thanks for coordinating this and hope to learn more soon,
Will
wpsternlicht@gmail.com

**Reply by kieranwint** [reply]
Hi @wps113 - thanks for registering your interest. We'll reach out when plans for the working group have been finalised!

**Comment by amackayy** [comment]
Hello! I work for VU.CITY. Our platform integrates high-precision 3D modelling and geospatial data, enabling users to visualise, simulate, and evaluate the impact of developments in a virtual environment. We work with numerous LPA's across the UK in implementing this technology into their workflows to help speed up and make the the planning process more transparent. We would like to be part of the working group - my email is a.mackay@vu.city 

Thanks!
Anna

**Reply by kieranwint** [reply]
Hi @amackayy - thanks for registering your interest. We'll reach out when plans have been finalised!

**Comment by AlistairWalker1** [comment]
Hi, I' Alistair Walker, an Environmental Planner at Quod. Datasets underpin Environmental Impact Assessments through all stages. I have done collaborative exploratory work in the space of digitising the EIA process before, with a standardised nationwide planning dataset seen as keen to underpin all processes. I would be keen to be involved in the discussion.

**Reply by kieranwint** [reply]
Hi @AlistairWalker1 - thanks for registering your interest. If you could please supply us with an email address we'll reach out when plans are finalised, thanks!

**Comment by AlistairWalker1** [comment]
Hi,

Please use this company email address: ***@***.******@***.***>


***@***.***



***@***.***<https://uk.content.exclaimer.net/?url=https%3A%2F%2Fwww.quod.com%2Fnews%2Fquod-wins-rtpi-planning-consultancy-of-the-year%2F&tenantid=3IGIroQiEeiAwgAVXYyXFw&templateid=d518d2e35740ea11a1cc0003ff00cbd6&excomponentid=lG-QAZmeQeMFRxI35ut9zUl8eDJfqwGdbapVk7bxjH0&excomponenttype=Image&signature=LDFZQvDaZINeZVju6EkmzSncy-0UJB-6_slhZOEQDz8X4xzDnTWv8LzpS5GsT0-iWoIvlepk4sj6B3gDQfGm0QCH5nxBSQmNyQ-lFIndJz2r2uUx7nsIUfLval02eyJwdRJq-MrxrhHqAlPRhtuFP2OvVhHbTEVfFtORboq1mtTATJhQ-ehCdY3YdgWfae5j88y5KTs_0wFIoEIAGHkX6eBSkGToGB3O52PmxMeF9ty9dDJRg8ktmQuyT60-aWvQP0_lL74T31wGVQKhTyvZEVlIdVJKmszsg46ULFbOJi7uLTp1JtyMSNRtdij6MeMxnCN2MWopiN76Ewjg9NaXVQ&v=1&imprintMessageId=1aca6433-39f4-4340-a2b7-3f632ed9c572>




Alistair Walker​​​​



Associate


***@***.******@***.***>






Mobile:


07834 176 254<tel:07834%20176%20254>




www.quod.com<https://uk.content.exclaimer.net/?url=http%3A%2F%2Fwww.quod.com%2F&tenantid=3IGIroQiEeiAwgAVXYyXFw&templateid=d518d2e35740ea11a1cc0003ff00cbd6&excomponentid=9j_g8QHtlTXVMfVMbiSUHEKZu9w_stYDfxgkPNVGEEE&excomponenttype=Link&signature=dn61mEn1_81f7ueCHcPR1P0p1q98EOEIJNnQy1xn519th5gLV0CHJIe3eCrKnmAYmm0InADJsjDr8oQ7kkM_JSY-mhHTkX0dWbT-31JKS-c12xirvCTV6tMT3T_99hJMc0-QwqvtsasgpiWIJJdb2kgD9DgWv-UZtLF1mwOuO4y35Si1ZtnJmfm1yY-NhqxFHyItb6CDlTIVwQ05dkkV1egK6kS0fkjeFes80QIfwxVjD9nOxoQ8EwdYwPw15fc01kgSyl4BGptJrP1xVS5G4-Qqh_qdyhiAFC3iHzYHNKq6c74UpkAwB8H2KYHB5JcDzQEs6NFbkWh4l8tR3lalRA&v=1&imprintMessageId=1aca6433-39f4-4340-a2b7-3f632ed9c572>




21 Soho Square



London



W1D 3QP







***@***.***

**Comment by EwaMoskwiak** [comment]
Hello, we are PlanningHub and we deploy AI technologies to access site and project specific planning information. We would like to be added to this project. Best wishes, Ewa ewa@planning-hub.com

**Reply by kieranwint** [reply]
Hi @EwaMoskwiak - thanks for showing your interest! We will add you to the invite list for the next community drop-in for planning application specifications, which is happening tomorrow morning. Hopefully you can make it!

**Reply by EwaMoskwiak** [reply]
Hi Kieran, that is great. would you be kind enough to confirm the time so I can move meetings to accommodate? Best wishes, Ewa

**Reply by kieranwint** [reply]
@EwaMoskwiak it's tomorrow at 11:00, hope that works for you!

**Comment by EwaMoskwiak** [comment]
Hi Kieran,

 

Wonderful, I have moved things around so I can attend. Thank you so much. Please feel free to send the invite.

 

Best wishes,

Ewa

**Reply by harryquartermain** [reply]
Can I get an invite to this drop in session please?

**Reply by kieranwint** [reply]
Hi @harryquartermain, sure thing - what is your email address?

**Reply by harryquartermain** [reply]
harry.quartermain@land.tech

**Comment by stevenbrandwood** [comment]
Hi all.  Steve from GeoPlace here to bang the drum for definitive addressing and Unique Property Reference Numbers and Unique Street Reference Numbers.  If you're not aware of what we do here is our website www.geoplace.co.uk

Essentially, we manage the national address index by receiving address and street updates from local authorities and publish via OS. The index (or key register) unambiguously records all addresses (whether they receive post or not), gives a geospatial location and a unique ID.  The data also contains parent child relationships, history and use classifications.

We are working with local authorities to improve the recording of early property lifecycle information in order to capture sites and plots in the planning stages and link this through to building control.  

I look forward to presenting what we do to this community as I think UPRNs and USRNs are vital to the success to digital planning.

**Reply by PRWhitehouse** [reply]
Hi Steve, the UPRN / USRN are essential to any forward digital planning initiatives. New exited professional looking in to digital planning should not forget this important foundation stone. Going back to the Local e-Government modernisation agenda, the e-Planning Programme, back then, could not achieved its outcomes without the National and Local Property Gazetteer. Good to see you (Steve) are still championing the cause.

**Comment by cday-hybrid** [comment]
[like]  Claire Day reacted to your message:
________________________________

**Comment by Alchemist-Ra** [comment]
Hi! I'm an LPA planner with a particular interest in digitisation, automation and AI and I would love to be involved, however I can help! 
Zoe McAden

**Reply by MrMikeRose** [reply]
Hi Zoe, so sorry for the delay in responding to you... here is our website, with a link to our sign up form for our regular community meetings https://design.planning.data.gov.uk/project/planning-applications

You will find my email in the recent notes from our community sessions if you want to ping me and we can catch up.

M
